libdokan/folderlist.go

```diff
@@ -86,7 +93,15 @@ func (fl *FolderList) open(ctx context.Context, oc *openContext, path []string)
 		}
 		fl.mu.Unlock()
 
+		h, er := libkbfs.ParseTlfHandle(
```

Comment:
hmm.. This line was moved to above several other calls, and `err` would be overridden by other statements.